src/main/java/com/github/pedbertor/gol/controller/ControllerBuilder.java

```diff
@@ -106,6 +107,17 @@ public ControllerBuilder setCellStateProcessor(CellStateProcessor cellStateProce
         return this;
     }
 
+    /**
+     * Sets console output state.
```

Comment:
@Sumlarys I would update the method description in the javadoc to be more precise with its intent (i.e. "Enable/disable console output").